build.gradle

```
@@ -264,6 +268,9 @@ allprojects {
         systemProperty 'postgresPassword', postgresPassword
         systemProperty 'org.slf4j.simpleLogger.showDateTime', true
         systemProperty 'org.slf4j.simpleLogger.dateTimeFormat', 'yyyy-MM-dd HH:mm:ss:SSS Z'
+        systemProperty 'sqlPort', sqlPort
+        systemProperty 'sqlUser',sqlUser
+        systemProperty 'sqlPassword',sqlPassword
```

I think these should match the properties set earlier.
```suggestion
        systemProperty 'mssqlPort', sqlPort
        systemProperty 'mssqlUser',sqlUser
        systemProperty 'mssqlPassword',sqlPassword
```